tests/network/vmi_networking.go

```diff
@@ -723,9 +723,18 @@ var _ = Describe("[Serial][rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][le
 				ipAddr := gatewayIPFromCIDR(networkCIDR)
 				Expect(libnet.PingFromVMConsole(serverVMI, ipAddr)).To(Succeed())
 
-				By("Checking ping (IPv4) to google")
-				Expect(libnet.PingFromVMConsole(serverVMI, "8.8.8.8")).To(Succeed())
-				Expect(libnet.PingFromVMConsole(clientVMI, "google.com")).To(Succeed())
+				By("Checking ping (IPv4)")
+				if len(flags.IPV4ConnectivityCheckAddress) != 0 {
+					Expect(libnet.PingFromVMConsole(serverVMI, flags.IPV4ConnectivityCheckAddress)).To(Succeed())
+				} else {
+					By("Skipping ping test. IPV4 address was not provided")
```

Comment:
I agree it is not. I removed it. 

Since now an env variable is used to tell if the tests should be skipped or not, I added a line to the log that is prompt when 
running the tests: 
https://github.com/kubevirt/kubevirt/blob/982b4f24b6ad4eabc458be1b0697deb526762c6a/hack/functests.sh#L52
stating the tests will be skipped.

@RamLavi  is this acceptable?